app/src/main/java/dev/arkbuilders/arkmemo/graphics/SVG.kt

```diff
@@ -65,65 +71,49 @@ class SVG {
         }
     }
 
-    fun getPaths() = createPaths()
+    fun getPaths() = paths
 
     fun copy() = SVG().apply {
         strokeColor = this@SVG.strokeColor
         fill = this@SVG.fill
         viewBox = this@SVG.viewBox
-        commandsArray = this@SVG.commandsArray
+        commandsArray.addAll(this@SVG.commandsArray)
+        paths.addAll(this@SVG.paths)
     }
 
-    private fun createPaths(): Stack<DrawPath> {
-        val paths = Stack<DrawPath>()
-        if (pathData.isNotEmpty()) {
+    private fun createPaths() {
+        commandsArray.addAll(pathData.split("$COMMA$SPACE"))
```

Comment:
Is it really should be split by `, ` (with space)? Can elements of a path be separated by plain `,` (without space)?

Also, would be great to have stages of SVG workflow clearly separated:
1. **Parsing** should result in abstract data structure without any strings, e.g. `List<SvgCommand>`.
2. **Mutation** of this data structure during the note modification. If we created new note, then we start with empty list.
3. **Printing** of the data structure into a file to save the note. More precisely, SVG/XML syntax generation.

Also, double-check that SVG parsing is performed only during notes loading.

Reply:
Yes, its possible to separate the commands by just comma